src/main/java/com/felixgrund/codeshovel/tasks/AnalysisTask.java

```diff
@@ -330,4 +331,10 @@ public StartEnvironment getStartEnv() {
 	public int getNumCommitsTotal() {
 		return numCommitsTotal;
 	}
+
+	public  void setFunctionAnnotation(String annotations) {
+		this.functionAnnotation = annotations;
+	}
+
+	public String getFunctionAnnotation() { return functionAnnotation; }
```

Comment:
sorry for the nitpick but can we also move this return statement to the next line if you're gonna be adding commits anyway?

Reply:
Oopsie sorry about that 👍🏾 Thanks